src/js/structs/PodInstance.js

```diff
@@ -44,6 +51,18 @@ module.exports = class PodInstance extends Item {
     }
   }
 
+  getLastChanged() {
+    return new Date(this.get('lastChanged'));
+  }
+
+  getLastUpdated() {
+    return new Date(this.get('lastUpdated'));
+  }
+
+  getName() {
```

Comment:
Here is the commit, if you want to revert it: https://github.com/dcos/dcos-ui/pull/1068/commits/ce12e9b10635e4c92415ed90dcf750505110b104